bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/GraalJSScriptEngineFactory.java

```diff
@@ -32,13 +30,24 @@
 @Component(service = ScriptEngineFactory.class)
 public final class GraalJSScriptEngineFactory implements ScriptEngineFactory {
 
+    public static final String MIME_TYPE = "x-application/javascript.new";
```

Comment:
```suggestion
    public static final String MIME_TYPE = "application/javascript;version=ECMAScript-2021";
```

As suggested by @jpg0 in another conversation